samples/node/Dockerfile

```diff
@@ -0,0 +1,2 @@
+FROM sk8s/node-function-invoker:latest
+ADD square.js /functions/node/square.js
```

Comment:
it seems like we could add this to a default location/name such as `/function.js` so that we don't need the `FUNCTION_URI` when that default is used